jwst/extract_1d/extract.py

```diff
@@ -2800,6 +2809,12 @@ def do_extract1d(
         reference file (or the default position, if there is no reference
         file) will be shifted to account for source position offset.
 
+    center_xy : int or None
```

Comment:
Might be a nitpick, but should this type def be list[int] or None, or iterable[int] or None, rather than int or None?

Reply:
No idea